ee/app/models/server/models/CannedResponse.ts

```diff
@@ -0,0 +1,83 @@
+import { DeleteWriteOpResultObject, FindOneOptions } from 'mongodb';
+import _ from 'underscore';
+
+import { Base } from '../../../../../app/models/server';
+import { IOmnichannelCannedResponse } from '../../../../../definition/IOmnichannelCannedResponse';
+/**
+ * Livechat Department model
+ */
+export class CannedResponse extends Base {
```

Comment:
Some of the methods here are duplicated within the raw model, so wondering if it would be safe to delete them from here?